WorldCupWrapped/Models/TrophyCount.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace WorldCupWrapped.Models
+{
+    public class TrophyCount
```

Comment:
Clasa asta nu e degeaba? gen daca avem Team, Trophy si Count-ul in clasa TeamTrophy, nu mergeau luate de acolo si sa fii fct un controller pt clasa TeamTrophies?

Reply:
nu mergeau, pt ca el initial imi returna un List<Trophy>, doar ca modelul nostru de Trophy nu are si un "count", si trebuia cumva sa fac un fel de join cu TeamTrophy ca sa vad fiecare echipa de cate ori a castigat

am mai incercat sa ii dau un List<(Trophy trophy, int count)>, dar nu exista o mapare pentru asta